testsuite/MDAnalysisTests/lib/test_util.py

```diff
@@ -651,40 +640,48 @@ def _check_get_reader(self, fn, R):
     def _check_get_reader_invalid(self, fn):
         assert_raises(ValueError, mda.coordinates.core.get_reader_for, fn)
 
+    @pytest.mark.parametrize('ext', map(lambda ext: ext.upper(),
+                                        map(lambda format: format[0], formats)) +
+                                        map(lambda ext: ext.lower(), map(lambda format: format[0], formats)
+    ))
+    def test_get_extention(self, ext):
```

Comment:
This is the new function. (FYI this will be done directly on `_check_get_ext` - this wrapper method will not be there in the final version).